### PR TITLE
Feature/search users

### PR DESCRIPTION
**Pull request by SadilekJ**

### CODE REVIEW

File: src/lib/pages/Profile.svelte [hunk 1]
```diff
@@ -36,6 +38,12 @@
       .collection(MongoCollections.Users);
     const result = (await data.find()) as UserData[];
     user = result.find((user) => user.personalId === realmApp.currentUser.id);
+    if (user != null) {
+      canSeePage = true;
+      if (user.accountState === 0) {
```
Comment (SedlarDavid) [inline src/lib/pages/Profile.svelte:43]: bleh porovnavat enum s intem.... co by na to rekl tom :D

File: src/lib/pages/Profile.svelte [hunk 2]
```diff
@@ -65,86 +72,86 @@
...
+              type="text"
+              id="surname"
+              placeholder="Surname"
+              required
+              bind:value={user.surname}
+            /></TableBodyCell
+          >
+          <TableBodyCell>
+            <Input
+              disabled={user.accountState == 1}
```
Comment (SedlarDavid) [inline src/lib/pages/Profile.svelte:119]: bleh podruhe

File: src/lib/pages/administration/Administration.svelte [hunk 3]
```diff
@@ -16,18 +16,66 @@
...
+  let canSeePage = false;
+
+  async function getCurrentUserInfo() {
+    const data = mongo.collection(MongoCollections.Users);
+    const result = (await data.find()) as UserData[];
+    currentUser = result.find(
+      (user) => user.personalId === realmApp.currentUser.id
+    );
+    if (currentUser != null) {
+      if (currentUser.role === 1) canSeePage = false;
```
Comment (SedlarDavid) [inline src/lib/pages/administration/Administration.svelte:55]: bleh potreti

**Review by SedlarDavid — APPROVED** [review]
budiz

File: src/lib/pages/Profile.svelte (hunk 1 above)
Comment (SadilekJ) [reply src/lib/pages/Profile.svelte:43]: Ten enum se tak ale zobrazuje a ukládá v tom mongu :D

File: src/lib/pages/Profile.svelte (hunk 2 above)
Comment (SadilekJ) [reply src/lib/pages/Profile.svelte:119]: Same as above

File: src/lib/pages/administration/Administration.svelte (hunk 3 above)
Comment (SadilekJ) [reply src/lib/pages/administration/Administration.svelte:55]: Same as above :D

**Pull request merged**